csharp/Platform.Setters/Setter[TResult].cs

```diff
@@ -5,32 +5,34 @@
 namespace Platform.Setters
 {
     /// <summary>
-    /// <para>
-    /// Represents the setter.
-    /// </para>
-    /// <para></para>
+    /// <para>Represents a setter that allows to set a passed value as the result. This setter variant has <typeparamref name="TDecision"/> preset to <see cref="Boolean"/>.</para>
+    /// <para>Представляет установщик, позволяющий устанавливать переданное ему значение в качестве результата. Для этого варианта установщика <typeparamref name="TDecision"/> предустановлен в <see cref="Boolean"/>.</para>
     /// </summary>
+    /// <typeparam name="TResult">
+    /// <para>The type of a result value.</para>
+    /// <para>Тип результирующего значения.</para>
+    /// </typeparam>
+    /// <typeparam name="TDecision">
+    /// <para>The type of values indicating true and false.</para>
+    /// <para>Тип значений обозначающих истину и ложь.</para>
+    /// </typeparam>
     /// <seealso cref="Setter{TResult, bool}"/>
```

Comment:
We should reference another type here. That means we have to use `TDecision` instead of `bool`.